documentation/index.md

```diff
@@ -0,0 +1,25 @@
+# Charmed OpenSearch Documentation
+The Charmed OpenSearch Operator delivers automated operations management from day 0 to day 2 on the [OpenSearch Community Edition](https://github.com/opensearch-project/OpenSearch/) document database. It is an open source, end-to-end, production-ready data platform product on top of juju.
+
+OpenSearch is a popular NoSQL database. It stores data as JSON documents in searchable indices, which are queried using a REST API. OpenSearch is also a distributed database, so vertical and horizontal scaling come naturally.
```

Comment:
It is not only a database, it is designed for search.
```suggestion
OpenSearch is an open-source search and analytics suite that makes it easy to ingest, search, visualize, and analyze data. The OpenSearch project is a forked search project based on old versions of Elasticsearch and Kibana. 
```

Reply:
Good suggestion, thanks!